### PR TITLE
Zhyr patch 1

### PR DESCRIPTION
**Pull request by zhyr**
提交merge分支

### CODE REVIEW

File: docs/assets/2019/15660383619044 [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (qhduan) [inline docs/assets/2019/15660383619044]: why there a file without ext?

**Comment by lijiarui** [comment]
@zhyr  There are two things you need to do next:

1. Please sign the CLA above (click the "CLA not signed yet" button in reply to your pull request)
2. Make the CI happy by following the error messages from CI Log

Please fix the above two issues before we publish your great post, and feel free to let me know if you have any questions.

**Comment by zhyr** [comment]
merge ,update for dir of image and post

**Comment by zhyr** [comment]
> @zhyr There are two things you need to do next:
> 
> 1. Please sign the CLA above (click the "CLA not signed yet" button in reply to your pull request)
> 2. Make the CI happy by following the error messages from CI Log
> 
> Please fix the above two issues before we publish your great post, and feel free to let me know if you have any questions.

ok，fix

**Comment by zhyr** [comment]
删除asset目录下md文件

**Pull request merged**